stdlib/scripts/install-build-tools-linux.sh

```diff
@@ -0,0 +1,17 @@
+LLVM_VERSION=17
```

Comment:
**Suggestion** (Here and in the other script), I recommend adding

```
#!/usr/bin/env bash
```

in case users want to invoke these directly.  Make sure they have user executable bit set too (`chmod +x stdlib/scripts/install-build-tools-linux.sh`).

Reply:
Also, adding `set -euo pipefail` in both scripts would be good for those that invoke the scripts directly too.

Reply:
Done. Thanks!